ors-engine/src/main/java/org/heigit/ors/isochrones/builders/fast/FastIsochroneMapBuilder.java

```diff
@@ -343,7 +335,7 @@ private List<Double> createCoordinateListFromGeometry(Geometry preprocessedGeome
                             ring.getPointN(i).getX(),
                             ring.getPointN(i + 1).getX(),
                             contourCoordinates,
-                            MIN_EDGE_LENGTH_LIMIT,
+                            minSplitLength/2,
```

Comment:
Why is this divided by two, and should we then directly pass the value already divided?

Reply:
This is so because the initial hope was that `minSplitLength` would be enouggh, however `minSplitLength`is is the correct value only for edges around the equator. I figured out later that the whole approach of splitting based on the length in meters is incompatible with how the concave hull is actually being computed in the geocoordinate domain. The current solution of taking half of the length is more of a heuristic which seems to work pretty well, a proper fix is underway in a separate PR 🤞